manifests/s/Splashtop/SplashtopStreamer/3.5.4.0/Splashtop.SplashtopStreamer.installer.yaml

```
@@ -13,7 +13,7 @@ InstallModes:
 InstallerSwitches:
   Silent: prevercheck /s /i hidewindow=1
   SilentWithProgress: prevercheck /s /i hidewindow=1
-UpgradeBehavior: install
+UpgradeBehavior: uninstallPrevious
```

```suggestion
UpgradeBehavior: install
AppsAndFeaturesEntries:
- InstallerType: msi
  ProductCode: '{B7C5EA94-B96A-41F5-BE95-25D78B486678}'
```
The upgrade failure is due to installer technology mismatch. The logs say that 
```
2023-02-12 11:52:34.091 [CLI ] Installer [X86,exe,Machine,en-US] not applicable: Installed package type 'msi' is not compatible with installer type exe
```
The application is actually an msi installer packaged into an exe. Setting the InstallerType to msi should resolve the issue as I've manually tested it in a VM.